src/expr/src/interpret.rs

```diff
@@ -626,12 +626,12 @@ impl Interpreter for Trace {
     ) -> Self::Summary {
         // TODO: this is duplicative! If we have more than one or two special-cased functions
         // we should find some way to share the list between `Trace` and `ColumnSpecs`.
-        let (left_monotonic, right_monotonic) = match func {
+        let (left_pushdownable, right_pushdownable) = match func {
             BinaryFunc::JsonbGetString { stringify: false } => (true, false),
             _ => func.is_monotone(),
         };
-        left.min(left_monotonic.into())
-            .max(right.min(right_monotonic.into()))
+        left.min(left_pushdownable.into())
```

Comment:
Urgh, now I get it, but I think this will be more readable if we use the `std::cmp::{min, max}` methods - something like this definitely makes things more evident:

```rust
max(
  min(left, left_pushdownable.into()),
  min(right, right_pushdownable.into()))
```